src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,150 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private int size;
+
```

Comment:
The task is not done

You have only one pointers.

![image](https://github.com/mate-academy/jv-linked-list/assets/109764850/0c3ccaf8-1f7c-4815-8c64-71beb71e0f37)